### PR TITLE
Add my data to public/data

### PR DESCRIPTION
**Pull request by Pulkit007**
Fixes #

## Proposed Changes

  -
  -
  -

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/152"></a>

### CODE REVIEW

**Review by FayasNoushad — APPROVED** [review]

**Review by Recognizeyourself — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
could you resolve the merge conflict? If you need any help please let us know

File: public/data/_list.json [hunk 1]
```diff
@@ -26,5 +26,6 @@
   { "username": "voidz", "avatar": "https://github.com/voidz7.png" },
   { "username": "suhail34", "avatar": "https://github.com/suhail34.png" },
   { "username": "vikasganiga05", "avatar": "https://github.com/vikasganiga05.png" },
-  { "username": "roshanmhatre", "avatar": "https://github.com/RoshanMhatre.png" }
+  { "username": "roshanmhatre", "avatar": "https://github.com/RoshanMhatre.png" },
+  { "username": "Pulkit007", "avatar": "https://github.com/Pulkit007.png" }
```
Comment (schmelto) [inline public/data/_list.json:30]: This one is not necessary anymore and will be updated automatically

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by eddiejaoude** [comment]
I will merge into a temporary branch and fix

**Comment by eddiejaoude** [comment]
Unfortunately it won't let me do it. So I have created a patch and I will add it manually to the `main` branch but I have to close this PR

**Comment by eddiejaoude** [comment]
Done. Here is the commit https://github.com/EddieHubCommunity/LinkFree/commit/35105ccd06dd77dd459f0514301cacff141f0b46

**Pull request closed without merging**